app/src/main/java/org/mozilla/fenix/compose/list/ListItem.kt

```diff
@@ -51,6 +51,7 @@ fun TextListItem(
     label: String,
     modifier: Modifier = Modifier,
     description: String? = null,
+    maxDescriptionLines: Int = 1,
```

Comment:
Can add a @param for this in `TextListItem ` and `ListItem` below? I would just use a similar description for `maxLines` in `SecondaryText` - `An optional maximum number of lines for the description text to span.`

Reply:
Thanks for the reminder!